lib/theme/colors.js

```diff
@@ -13,6 +13,7 @@ const ocBrandColors = {
   200: '#C2E2FF',
   100: '#E6F3FF',
   50: '#F5FAFF',
+  20: '#3385FF',
```

Comment:
We don't want to change the design system. As per https://github.com/opencollective/opencollective-frontend/pull/5887#discussion_r576334939:

> No need to create a new one, as mentioned in the issue description we should replace them with existing colors from the design system.

```suggestion
```